src/modules/Lan.rb

```diff
@@ -370,7 +370,10 @@ def Read(cache)
       # Progress step 6/9
       ProgressNextStage(_("Reading hostname and DNS configuration...")) if @gui
       DNS.Read
-      Host.Read
+      if !Host.Read
```

Comment:
`yast2 host` will fail because it does not check the return value or Host.Read, and there are other callers too.